protocol/v2/ssv/validator/validator.go

```diff
@@ -81,7 +77,7 @@ func NewValidator(pctx context.Context, cancel func(), options Options) *Validat
 		role := dutyRunner.GetBaseRunner().RunnerRoleType
 
 		v.Queues[role] = queueContainer{
-			Q: queue.WithMetrics(queue.New(options.QueueSize), options.Metrics),
+			Q: queue.New(options.QueueSize),
```

Comment:
why no metrics here? in committee you added the metrics for the queue.

Reply:
@moshe-blox not talking about anything specific, just wondered why the change

Reply:
@y0sher that metric was not used on any of the dashboards, also it was disabled in production and it was high cardinality (msgID as a label) hence it was removed.